app/create_app.py

```diff
@@ -110,13 +113,27 @@ def ensure_minimum_required_roles():
                 unprotected_routes=["/", "/healthcheck"],
             )
 
+        static_files_list = []
+        for _, _, files in os.walk(os.getcwd() + "/app/static"):
```

Comment:
want to use `STATIC_URL_PATH` here instead of hardcoded. But the value set to `STATIC_URL_PATH = "/assets"` in config/default.py which seems to be wrong

Reply:
we could change this location maybe?

Reply:
done